### PR TITLE
Add gradient check to reduce_max and reduce_min ops

### PR DESCRIPTION
**Pull request by qjing666**
fix OP with no grad_check：

- reduce_max: add grad_check

- reduce_min: add grad_check

### CODE REVIEW

**Comment by qjing666** [comment]
reduce_max and reduce_min are discontinuous non-derivable functions, their gradient check are not supported by unittest framework. So add skip_check_grad_ci.

File: python/paddle/fluid/tests/unittests/test_reduce_op.py [hunk 1]
```diff
@@ -51,6 +51,9 @@ def test_check_grad(self):
         self.check_grad(['X'], 'Out')
 
 
+@skip_check_grad_ci(
+    reason="reduce_max is discontinuous non-derivable function,"
+    " its gradient check is not supported by unittest framework.")
 class TestMaxOp(OpTest):
```
Comment (zhangting2020) [inline python/paddle/fluid/tests/unittests/test_reduce_op.py:57]: 应该还有两个case：TestReduceMaxOpMultiAxises、TestReduceMinOpMultiAxises，这两个也需要进行修改

Comment (qjing666) [reply python/paddle/fluid/tests/unittests/test_reduce_op.py:57]: done

**Review by zhangting2020 — APPROVED** [review]

**Review by zhangting2020 — DISMISSED** [review]
LGTM

**Review by luotao1 — APPROVED** [review]

**Pull request merged**